src/components/VFilters/VLicenseExplanation.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="license-explanation w-full max-w-xs p-6">
+    <h5 class="text-base font-semibold">
+      <template v-if="isTechnicallyLicense">{{
+        $t('filters.license-explanation.license-definition')
+      }}</template>
+      <template v-else>{{
+        $t('filters.license-explanation.mark-definition', {
+          mark: license.toUpperCase(),
+        })
+      }}</template>
+    </h5>
+
+    <VLicenseElements
```

Comment:
We should mark these as `aria-hidden` in this use case, otherwise the popover is extremely repetitive, saying the license name over and over again with no explanation why.

Reply:
Made ARIA hidden in [3f91c0e](https://github.com/WordPress/openverse-frontend/pull/850/commits/3f91c0e3354840c516c7026d2d8ee03e8ba99ae3).

Reply:
Sorry, when I made this comment I didn't realize that `VLicenseElements` rendered the explanation in addition to the icon. I totally misread this component, my fault.

We should remove `aria-hidden` from this. My other comment in this review addresses the specific concern I had.

Reply:
@dhruvkb @sarayourfriend is this resolved?